keyboards/crkbd/keymaps/nimishgautam/config.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
```

Comment:
Please add a GPLv2 compatible copyright notice to this file.

Reply:
Added spdx gpl lines. Thanks for marking the specific files needed as well so I know next time :)